tests/view_tests/tests/test_debug.py

```diff
@@ -258,7 +258,8 @@ def test_template_exceptions(self):
             except Exception:
                 raising_loc = inspect.trace()[-1][-2][0].strip()
                 self.assertNotEqual(
-                    raising_loc.find("raise Exception('boom')"), -1,
+                    raising_loc.find('raise Exception("boom")'),
```

Comment:
Matching the exact string, which (clearly) won't get updated. 🤔

Reply:
Yes, an exact sting in a trace, so it didn't match anymore when black updates `'boom'` to `"boom"` in the template tag.